core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -287,6 +308,48 @@ oppia.controller('StateInteraction', [
                 stateCustomizationArgsService.displayed = {};
               };
 
+              $scope.getSaveInteractionButtonStyleTooltip = function() {
+                if (!$scope.hasCustomizationArgs) {
+                  return 'No customization arguments';
+                }
+                if (!$scope.stateInteractionIdService.displayed) {
+                  return 'No interaction being displayed';
+                }
+
+                var warningsList = $scope.getCustomizationArgsWarningsList();
+                var warningMessages = warningsList.map(function(warning) {
+                  return warning.message;
+                });
+
+                if (warningMessages.length === 0) {
+                  if ($scope.form.schemaForm.$invalid) {
+                    return 'Some of the form controls failed validation';
```

Comment:
Bit technical. Suggest: "Some of the form entries are invalid." or "Please fix the invalid form entries."